### DISCUSSION TITLE
[Feature Request] Entry field default selection

### DISCUSSION THREAD

**Discussion by TheFunkyMonk** [craftcms/cms#9032] [Ideas]
### Description

We have a pretty common setup for creating new landing pages in most of our sites where the author can add a hero, a set of "modules" from a Matrix-style block field, and in this case, a large CTA at the bottom of the page that contains some copy and a link to a different piece of content.

![Screen Shot 2021-04-30 at 2 03 19 PM](https://user-images.githubusercontent.com/1728770/116754264-449d3a80-a9bd-11eb-9931-92b152db593b.png)

Since the author is going to use the same CTA 90% of the time, we've set the CTA field up as an entry selector for a "Footer CTAs" channel containing an entry containing the CTA content so they don't need to re-write the headline/copy for the CTA every time they want to use the default one.

By default, we'd like all new landing pages to link to the entry we've already created in this channel while giving the author the option to either remove it or replace it with a different entry they've created in the "Footer CTAs" channel.

## Current Behavior

Currently, the only ways to accomplish this default selected entry behavior are:
- Hardcode the default entry into the template
- Create a global field to set as the default entry

Both of these are a little more confusing from an authoring perspective, and would require an additional lightswitch toggle to disable/hide the CTA if they want a page to not include a CTA at all.

## Requested Behavior

Entry fields support a default entry selection. 

![entry-select-default](https://user-images.githubusercontent.com/1728770/116755992-3270cb80-a9c0-11eb-93fc-cf20e1977ee9.jpg)

This way, we could eliminate the need to manually select an entry every time a new page is created, as well as clearly indicate when an entry should not be shown.

**Comment by brandonkelly** [comment]
This would be tricky since field settings are stored in the project config, but individual entries are not. Not really sure how we’d do it.
